backend/tests/endpoints/test_message.py

```diff
@@ -158,8 +205,109 @@ async def test_send_message_check_status_code(mock_get_user_room, mock_write):
     """
 
     write_response = {"status_code": 422, "message": "unprocessible error"}
-    mock_get_user_room.return_value = fake_core_data
+    mock_get_user_room.return_value = fake_core_room_data
     mock_write.return_value = write_response
-    response = client.post(send_message_test_url, json=test_payload)
+    response = client.post(send_message_test_url, json=send_message_test_payload)
     assert response.status_code == 424
     assert response.json() == {"detail": {"Message not sent": write_response}}
+
+
+@pytest.mark.asyncio
+async def test_update_message_sucessful(
+    mock_get_message, mock_update_message, mock_centrifugo
+):
+    """Update message successful.
+
+    Args:
+        mock_get_message (AsyncMock): Asynchronous external api call
+        mock_update_message (AsyncMock): Asynchronous external api call
+        mock_centrifugo (AsyncMock): Asynchronous external api call
+    """
+    mock_get_message.return_value = fake_zc_core_message_data
+    mock_update_message.return_value = {
+        "status": 200,
+        "message": "success",
+        "data": {"matched_documents": 1, "modified_documents": 1},
+    }
+    mock_centrifugo.return_value = {"status_code": 200}
+    response = client.put(update_message_test_url, json=update_message_test_payload)
+    assert response.status_code == 200
+    assert response.json() == {
```

Comment:
This test fails because of `edited` value in `response.json()`. It should be True as the message has been successfullt edited, right?

Reply:
Yes boss @zxenonx but it is returning False in the endpoint.

Reply:
When we'll merge the Pagination PR. We will then fix this.

Reply:
Okay boss